source/includes/_campaign_quota.md

```diff
@@ -10,6 +10,7 @@ Create a quota for a specific campaign.
 {
   "num_respondents": 32,
   "name":"Age quota",
+  "is_click_balanced": 0,
```

Comment:
This should be `true` instead of `0`.  `1` and `0` work but we're returning a boolean, so it may be confusing to have an int here